data/core/units.cfg

```diff
@@ -247,7 +247,24 @@ The Dunefolk’s inquisitive and explorative nature does not preclude military s
         male_name= _ "race^Saurian"
         female_name= _ "race+female^Saurian"
         plural_name= _ "race^Saurians"
-        description= "" # wmllint: ignore
+        description= "Saurians are small, slender, and lizard-like creatures. Much smaller than humans, they rarely stand taller than a ten year old child, though the distance from the tip of their snout to the end their tail can be as long as the average man is tall. Light and nimble, the warriors prefer to fight as they hunt- slipping through enemy lines to target the weak and the injured while their slender figures allow them to evade their attackers.
+
+<header>text='Society'</header>
+Saurians are very mysterious creatures due to their tendency to live in areas inhospitable to others, such as swamps. Fatalistic in the extreme, Saurians believe all the events in a life can be predicted by the use of a complex form of astrology.
+
+Saurian culture is sharply segregated between the genders. Within each gender the members compete and through skill, determination, and reputation establish a clear pecking order, with a chief at the top. On those occasions when the two genders interact they do not contest for dominance, the situation determines the dominant gender. The chief of the males is alpha if they are within their village or encampment while the chief of females is dominant anywhere else. This continues down the rank structure with each male or female being dominant over any member of the opposite gender with lower rank and submitting to members of the opposite gender with higher rank.
```

Comment:
Comma splice at "dominance,".

"if they are within their" - ambiguous pronoun (could refer either to the males or to the tribe). "...is alpha within the clan's village or encampment..." maybe?

Reply:
"compete and through skill..." seems weird. I'd add a comma or semicolon before the "and" here to make it more clear that we're starting a new clause.

Reply:
That sentence sounds good to me. In everyday English I'd have put the verb "establish" before its modifier phrase "through skill, determination, and reputation", but Wesnoth doesn't tend to use everyday English :P